src/generic_mtrie.hpp

```diff
@@ -0,0 +1,116 @@
+/*
```

Comment:
Why are generic_mtrie.hpp and generic_mtrie_impl.hpp shown as new files in this commit? They already exist in the master. Interestingly, no conflict with the master is shown.

Reply:
merge commit squashed I'd guess